### DISCUSSION TITLE
Quartz notIndexed

### DISCUSSION THREAD

**Discussion by liwei19920225** [mtex-toolbox/mtex#1680] [Ask Anything]
Hi,
I face some problems on indexing my granitic samples using MTEX. In AZtec, phase image shows that quartz was well indexed with red color as below:
![EBSD 相颜色 Map_002](https://user-images.githubusercontent.com/119841643/205587200-e54596b4-beac-489b-8de3-b52a20dac8ff.jpeg)
But I get the phase image below using MTEX:
![untitled](https://user-images.githubusercontent.com/119841643/205588259-1f890387-a3ce-4673-b782-92fa9d139848.jpeg)
It shows that most quartz was labelled as notIndexed in MTEX.
My script:
% crystal symmetry
CS = {... 
  'notIndexed',...
  crystalSymmetry('-3m1', [4.9 4.9 5.4], 'X||a*', 'Y||b', 'Z||c*', 'mineral', 'Quartz', 'color', [0.53 0.81 0.98]),...
  crystalSymmetry('-1', [8.2 13 14], [93.508,115.908,90.658]*degree, 'X||a*', 'Z||c', 'mineral', 'Bytownite', 'color', [0.56 0.74 0.56]),...
  crystalSymmetry('12/m1', [8.5 13 7.2], [90,115.905,90]*degree, 'X||a*', 'Y||b*', 'Z||c', 'mineral', 'Orthoclase', 'color', [0.85 0.65 0.13]),...
  crystalSymmetry('12/m1', [5.4 9.3 10], [90,100.222,90]*degree, 'X||a*', 'Y||b*', 'Z||c', 'mineral', 'Biotite', 'color', [0.94 0.5 0.5]),...
  crystalSymmetry('12/m1', [5.2 9 10], [90,101.83,90]*degree, 'X||a*', 'Y||b*', 'Z||c', 'mineral', 'Muscovite 1M, magnesian', 'color', [0 0 0.55]),...
  crystalSymmetry('-3m1', [16 16 7.1], 'X||a*', 'Y||b', 'Z||c*', 'mineral', 'Tourmaline', 'color', [0.55 0 0])};

% plotting convention
setMTEXpref('xAxisDirection','east');
setMTEXpref('zAxisDirection','outOfPlane');
% create an EBSD variable containing the data
ebsd = EBSD.load(fname,CS,'interface','ctf',...
  'convertEuler2SpatialReferenceFrame');
plot (ebsd)
The output of unique(ebsd.phase) is:
ans =

     0
     2
     3
     4
     5
     6
     7
I think there is something wrong with the phase IDs.  Could someone give me some suggestions?
Thanks in advance.

**Comment by kilir** [comment]
Hi,
how do you set up your variable `CS`? Using the import wizard?
Cheers,
Rüdiger

**Reply by liwei19920225** [reply]
Thank you for reply. I use the import wizard to set up CS. The CS in CTF is:
Phases	7
4.915;4.915;5.406	90.000;90.000;120.000	Quartz	7	154			
8.178;12.870;14.188	93.508;115.908;90.658	Bytownite	1	2			
8.516;13.023;7.206	90.000;115.905;90.000	Orthoclase	2	12			
5.377;9.308;10.283	90.000;100.222;90.000	Biotite	2	12			
5.186;8.952;10.120	90.000;101.830;90.000	Muscovite 1M, magnesian	2	5			
5.199;9.005;10.164	90.000;101.300;90.000	Muscovite 1M, magnesian	2	12			
15.963;15.963;7.148	90.000;90.000;120.000	Tourmaline	7	160

**Reply by kilir** [reply]
Can you try and assign your 2nd Muscovite a uniquely defined name in the ctf?

**Reply by liwei19920225** [reply]
The question is solved. Thanks a lot.

**Reply by kilir** [reply]
Maybe you could quickly write what the issue was, so it might be helpful for others .
Cheers,
Rüdiger
